ccd-definition/ComplexTypes/StatementOfTruth.json

```diff
@@ -11,7 +11,7 @@
     "ID": "StatementOfTruth",
     "ListElementCode": "name",
     "FieldType": "Text",
-    "ElementLabel": "Name",
+    "ElementLabel": "Full Name",
```

Comment:
Just to confirm, this will change all instances of the Statement Of Truth - Name will be changed to Full Name across all instances. Just to check that is expected?

Reply:
I'll confirm this with BA

Reply:
QA has confirmed that if this is impacting all the Statement Of Truth pages, then this is expected and is ok.

Reply:
![image](https://user-images.githubusercontent.com/12885268/135104992-1eaf6abe-8aad-4ec6-a837-7cefce525a37.png)